src/terminal/parser/InputStateMachineEngine.cpp

```diff
@@ -364,7 +364,9 @@ bool InputStateMachineEngine::ActionCsiDispatch(const VTID id, const VTParameter
     // client reads it.
     if (_pDispatch->IsVtInputEnabled() &&
         _pfnFlushToInputQueue &&
-        id != CsiActionCodes::Win32KeyboardInput)
+        id != CsiActionCodes::Win32KeyboardInput &&
+        id != CsiActionCodes::FocusIn &&
+        id != CsiActionCodes::FocusOut)

```

Comment:
I'm assuming this filter is due to malicious.exe?
Could you copy the PR description here? This seems rather important to document in code.
(Well most things "should" be documented in code, but this one in particular sounds very important.)

Reply:
This actually I think is supposed to be in the parent branch. I think I noticed it here, but committed it with some other stuff instead of making it it's own commit and cherry-picking to the parent. I'll clean that up (and add comment)